package.json

```diff
@@ -64,7 +64,6 @@
     "@types/babel__core": "^7.20.0",
     "@types/eslint": "^8.40.0",
     "@types/node": "^20.2.3",
-    "@types/prettier": "^2.7.2",
```

Comment:
I found that this types package was just providing outdated types for things and getting in the way. prettier package itself seems to have types